h2o-core/src/main/java/water/rapids/ShuffleVec.java

```diff
@@ -0,0 +1,83 @@
+package water.rapids;
+
+import water.MRTask;
+import water.exceptions.H2OIllegalArgumentException;
+import water.fvec.*;
+import water.parser.BufferedString;
+import water.util.FrameUtils;
+import water.util.RandomUtils;
+
+import java.util.Random;
+
+import static water.util.RandomUtils.getRNG;
+
+public class ShuffleVec {
```

Comment:
Are you using this class somewhere? It is confusing; there is also another ShuffleTask... Please, keep only the final version.